vars/RedHat.yml

```diff
@@ -0,0 +1,14 @@
+---
+setup_packages:
+  - epel-release
+  - centos-release-scl
+  - centos-release-scl-rh
+  - gcc
+  - make
+  - glibc-devel
+  - kernel-devel
+python_packages:
+  - python
+  - python2-pip
+  - python-setuptools
+  - python-virtualenv
```

Comment:
Per comment regarding python installation task, we need to validate this location is where we want these to live.

Reply:
* https://github.com/elastic/ansible-elasticsearch/tree/master/vars
 * https://github.com/geerlingguy/ansible-role-postgresql/tree/master/vars

It seems fairly standard practice to put these vars here and there is a way to overwrite the variable used.